### DISCUSSION TITLE
Accessing data in the Qingping format

### DISCUSSION THREAD

**Discussion by ethicc** [custom-components/ble_monitor#1088] [Q&A] [answered]
I am interested in understanding how data transmitted in the Qingping format can be accessed.  When I use a BLE scanning tool and can see that the Qingping device I have (CGDN1) has six characteristics, three of which are write/read, on the UUID 22210000-554A-4546-5542-46534450464D. However, when I try to read the data using the read characteristic, nothing appears. Can anyone provide guidance on how to access this data and whether something needs to be written to the CGDN1 characteristic before it can be read? It is not apparent to me how ble_monitor does this.
Thanks
D

**Comment by Ernst79** [comment, marked as answer]
BLE monitor doesn’t read characteristics, it just scans passively for BLE advertisement which are emitted every x seconds. So there is no need to read/write characteristics. We use passive scanning 

We use aioblescan to do the passive scanning.
